Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -1344,6 +1391,8 @@ export function extend(publicAPI, model, initialValues = {}) {
   model.bgImage.style.zIndex = '-1';
 
   model._textureResourceIds = new Map();
+  model._graphicsResources = new Map();
+  model._graphicsResourceHash = new Map();
```

Comment:
This is subjective but you probably don't need a separate `Map` for hash strings. Maybe just add that as a part of value in _graphicsResources map.
`_graphicsResources.set( vtkObj, { gObj: resource, hash: hashStr });`

Reply:
That's a good suggestion. Leaving it as is for now but will modify when I revisit this.